cluster/gce/gci/configure-helper.sh

```diff
@@ -1840,6 +1840,16 @@ function update-node-label() {
   done
 }
 
+# A helper function that sets file permissions for kube-controller-manager to
+# run as non root.
+function run-kube-controller-manager-as-non-root {
+  prepare-log-file /var/log/kube-controller-manager.log ${KUBE_CONTROLLER_MANAGER_RUNASUSER} ${KUBE_CONTROLLER_MANAGER_RUNASGROUP}
+  setfacl -m u:${KUBE_CONTROLLER_MANAGER_RUNASUSER}:r "${CA_CERT_BUNDLE_PATH}"
+  setfacl -m u:${KUBE_CONTROLLER_MANAGER_RUNASUSER}:r "${SERVICEACCOUNT_CERT_PATH}"
```

Comment:
Are these shared files? If not, maybe just `chown`?

Reply:
Yes, these are shared files.